Gemfile

```
@@ -3,3 +3,7 @@
 source 'https://rubygems.org'
 
 gem 'rubocop', '~> 1.25', require: false
+gem 'rspec', '~> 3.11'
```

Let's start adding groups in the Gemfile.
```suggestion
group :development do
  gem 'rubocop', '~> 1.25', require: false
  gem 'rubocop-rspec', '~> 2.9'
end

group :test do
  gem 'factory_bot', '~> 6.2'
  gem 'faker', '~> 2.20'
  gem 'rspec', '~> 3.11'
end
```

---

Please add this rule to `rubocop.yml`

Style/StringLiterals:
  EnforcedStyle: double_quotes

```